index.js

```diff
@@ -129,13 +130,12 @@ module.exports = function (newman, options) {
       newman.exports.push({
         name: 'json-steps-reporter',
         default: 'newman-step-results.json',
-        path: options.jsonStepsExport,
+        path: options.xrayJsonExport,
         content: createLightSummary(o.summary, options)
```

Comment:
what i would do here is based on options passed from cli i will call createXraySummary function. So by default it calls createLightSummary but if my options says the export is for xray then createXraySummary is called. 
This way, i can add more exporters in future.